src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/customerSegment-detail.js

```diff
@@ -15,15 +14,13 @@ angular.module('virtoCommerce.demoSolutionFeaturesModule')
                 return false;
             };
 
-            $scope.setForm = (form) => { formScope = form };
-
             $scope.mainParameters = function () {
                 const parametersBlade = {
                     id: "mainParameters",
                     title: "demoSolutionFeaturesModule.blades.customer-segments-parameters.title",
                     subtitle: 'demoSolutionFeaturesModule.blades.customer-segments-parameters.subtitle',
-                    controller: 'virtoCommerce.demoSolutionFeaturesModule.customerSegmentsParametersController',
-                    template: 'Modules/$(virtoCommerce.DemoSolutionFeaturesModule)/Scripts/blades/mainParameters.tpl.html',
+                    controller: 'virtoCommerce.demoSolutionFeaturesModule.customerSegmentsMainParametersController',
```

Comment:
Singular, not plural. Here and below
`customerSegments` → `customerSegment`

Reply:
done